### PR TITLE
Add native_template_example to samples

### PR DESCRIPTION
**Pull request by malandr2**
## Description

Added `native_template_example` project to the samples folder.

Templates are sized with an aspect ratio for medium and small template types. 
Refresh Ad button is made available to see multiple ads in the same session.

## Related Issues

*Replace this paragraph with a list of issues related to this PR from the [issue database](https://github.com/googleads/googleads-mobile-flutter/issues). Indicate, which of these issues are resolved or fixed by this PR.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`).
This will ensure a smooth and quick review process. Updating the `pubspec.yaml` and changelogs is not required.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] My PR includes unit or integration tests for *all* changed/updated/fixed behaviors (See [Contributor Guide]).
- [x] All existing and new tests are passing.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] The analyzer (`flutter analyze`) does not report any problems on my PR.
- [x] I read and followed the [Flutter Style Guide].
- [x] I signed the [CLA].
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate a breaking change in CHANGELOG.md and increment major revision).
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/googleads/googleads-mobile-flutter/issues
[Contributor Guide]: https://github.com/googleads/googleads-mobile-flutter/blob/master/CONTRIBUTING.md
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[CLA]: https://cla.developers.google.com/

### CODE REVIEW

File: samples/admob/native_template_example/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# native_template_example
+
+An example project that demonstrates loading and showing native ads using native tempalates.
```
Comment (jjliu15) [inline samples/admob/native_template_example/README.md:3]: templates

File: samples/admob/native_template_example/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+When building and testing your apps, make sure you use test ads rather than
+live, production ads. Failure to do so can lead to suspension of your account.
+
+## Implementation
+
+The main steps to integrate rewarded ads are:
+
+1. Load an ad
+2. Handle ad events
+3. Present an into screen
```
Comment (jjliu15) [inline samples/admob/native_template_example/README.md:16]: *ad

File: samples/admob/native_template_example/README.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+### Load an ad
+The sample shows how to load a native ad.
+
+```
+_nativeAd = NativeAd(
+        adUnitId: _adUnitId,
+        listener: NativeAdListener(
+          onAdLoaded: (ad) {
+            print('$NativeAd loaded.');
```
Comment (jjliu15) [inline samples/admob/native_template_example/README.md:27]: Can you add a setState() call here that updates some instance variable `_nativeAdIsLoaded`?

File: samples/admob/native_template_example/lib/main.dart [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+                size: 16.0),
+            tertiaryTextStyle: NativeTemplateTextStyle(
+                textColor: Colors.black,
+                style: NativeTemplateFontStyle.normal,
+                size: 16.0)))
+      ..load();
+  }
+
+  void _loadVersionString() {
+    MobileAds.instance.getVersionString().then((value) {
```
Comment (jjliu15) [inline samples/admob/native_template_example/lib/main.dart:128]: Is displaying the version string intentional? Is this something that should be done in other samples as well?

File: samples/admob/native_template_example/README.md (hunk 1 above)
Comment (malandr2) [reply samples/admob/native_template_example/README.md:3]: Where should that be added?

Comment (malandr2) [reply samples/admob/native_template_example/README.md]: Done.

File: samples/admob/native_template_example/README.md (hunk 2 above)
Comment (malandr2) [reply samples/admob/native_template_example/README.md]: Done.

File: samples/admob/native_template_example/README.md (hunk 3 above)
Comment (malandr2) [reply samples/admob/native_template_example/README.md:29]: Done.

File: samples/admob/native_template_example/lib/main.dart (hunk 4 above)
Comment (malandr2) [reply samples/admob/native_template_example/lib/main.dart:128]: Yup, this is by design. It's done to match the same look and feel of the native_example for iOS and Android. (Only native displays the version string)

Comment (jjliu15) [reply samples/admob/native_template_example/lib/main.dart:128]: Got it, thanks!

**Review by jjliu15 — APPROVED** [review]

**Review by luo86 — APPROVED** [review]

**Pull request merged**